pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
```

Comment:
I just realized that you might be referring to the fact that I'm calling this from prereqs.py which isn't used in production. I'm going to remove that call as it was just a hack to try to get the Jasmine tests to run on Jenkins, and it didn't work anyway. I'll find a cleaner way to ensure that the vendor files are copied before the Jasmine tests are run on Jenkins.